serveradmin/common/templates/opensearch.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+<ShortName>ServerAdmin</ShortName>
+<Description>Server shell</Description>
+<Image width="16" height="16" type="image/x-icon">/favicon.ico</Image>
+<Url type="text/html" method="GET" template="{{ SCHEMA_FQDN }}/servershell/?term={searchTerms}">
```

Comment:
What is `SCHEMA_FQDN`?

Reply:
this is really draft, I would like to discuss how to do it properly